src/locale/locales/fr/messages.po

```diff
@@ -1555,23 +1536,23 @@ msgstr "J’ai un code"
 
 #: src/view/com/modals/VerifyEmail.tsx:216
 msgid "I have a confirmation code"
-msgstr ""
+msgstr "J’ai un code de confirmation"
 
 #: src/view/com/modals/ChangeHandle.tsx:283
 msgid "I have my own domain"
 msgstr "J’ai mon propre domaine"
 
 #: src/view/com/lightbox/Lightbox.web.tsx:158
 msgid "If alt text is long, toggles alt text expanded state"
-msgstr ""
+msgstr "Si le texte alternatif est trop long, affiche le texte alternatif complet"
```

Comment:
I don't really understand what it does, but toggle means there's a state switch so I gues it would be better to translate it ", change son mode d'affichage"